programmers/pull-two-out-and-add.js

```diff
@@ -0,0 +1,15 @@
+function solution(numbers){
+    const numbers_len = numbers.length;
```

Comment:
소문자로 쓰실거면 캐멀케이스를 이용해주세요
```suggestion
    const numbersLen= numbers.length;
```